rust/src/dart_bridge/api.rs

```diff
@@ -31,18 +33,22 @@ pub fn nokhwa_open_camera(friendly_name: String) -> usize {
     Box::into_raw(camera_box) as usize
 }
 
-pub fn nokhwa_set_camera_callback(camera_ptr: usize, operations: Vec<ImageOperation>, new_frame_event_sink: StreamSink<RawImage>) {
+pub fn nokhwa_set_camera_callback(camera_ptr: usize, operations: Vec<ImageOperation>, new_frame_event_sink: StreamSink<LiveCameraFrame>) {
     unsafe { 
         let mut camera = Box::from_raw(camera_ptr as *mut CallbackCamera);
         nokhwa::set_camera_callback(&mut camera, move |raw_frame| {
             if !is_flutter_app_alive() {
+                SKIPPED_LIVE_CAMERA_FRAMES.fetch_add(1, Ordering::SeqCst);
```

Comment:
And finally the back-end will only skip frames if the front-end is paused, so this should be 0 under normal circumstances in production.

Reply:
Indeed, this should be 0 under normal circumstances. It's just when the Flutter main isolate wasn't able to tell Rust it's still running due it being very busy with other stuff or the Flutter app is paused (e.g. by the debugger)